INSTALL.md

```diff
@@ -214,24 +230,24 @@ image. We'll use
 ### Linux
 
 ```sh
-.../cryptol-course> cryptol labs/Demos/OneTimePad.md
+.../cryptol-course$ cryptol labs/Demos/OneTimePad.md
 ...
 
 Loading module Cryptol
 Loading module labs::Demos::OneTimePad
 labs::Demos::OneTimePad> 
 ```
 
-Alternatively, you can use the `:m` or `:l` command from within 
+Alternatively, you can use the `:module` or `:load` command from within 
```

Comment:
Well, or set the `CRYPTOLPATH` environment variable to point to the root of the repository. Might be worth mentioning.

Reply:
Added CRYPTOLPATH info.